File: python/fizzbuzz/fizzbuzz.py
# fizzbuzz.py

class FizzBuzz:

  # fizz  = lambda self, x: "Fizz"     if type(x) is int and x %  3 == 0 else x
  fizz  = lambda self, x: "Fizz"     if x %  3 == 0 else x
  buzz  = lambda self, x: "Buzz"     if x %  5 == 0 else x
  fibu  = lambda self, x: "FizzBuzz" if x % 15 == 0 else x
  count = lambda self, x: list(range(1, x + 1))

  def play(self, x):
    numbers = self.count(x)
    return list(map(lambda x: self.fizz(self.buzz(self.fibu(x))), numbers))

File: js/fizzbuzz/fizzbuzz.js
const counter = x => {
  let output = [];
  for (let i = 1; i <= x; i += 1) {
    output.push(i);
  }
  return output;
};

const fizz = (x) => {
  if ( x % 3 == 0 ) {
    return 'Fizz';
  } else {
    return x;
  }
};

const buzz = (x) => {
  if ( x % 5 == 0 ) {
    return 'Buzz';
  } else {
    return x;
  }
};

const fibu = (x) => {
  if ( x % 15 == 0 ) {
    return 'FizzBuzz';
  } else {
    return x;
  }
};

const play = x => {
  let output = [];
  let numbers = counter(x);
  for ( let i = 0; i < numbers.length; i += 1 ) {

  }
};

module.exports = {
  counter,
  fizz,
  buzz,
  fibu,
  play
};